[2024-12-16T07:22:27.873+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-16T07:22:27.881+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_9843500346_2024-12-16T07-22-22-743313Z.wait_until_call scheduled__2024-12-16T07:22:22.743313+00:00 [queued]>
[2024-12-16T07:22:27.885+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_9843500346_2024-12-16T07-22-22-743313Z.wait_until_call scheduled__2024-12-16T07:22:22.743313+00:00 [queued]>
[2024-12-16T07:22:27.885+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-16T07:22:27.890+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): wait_until_call> on 2024-12-16 07:22:22.743313+00:00
[2024-12-16T07:22:27.895+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=117605) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-16T07:22:27.896+0000] {standard_task_runner.py:72} INFO - Started process 117606 to run task
[2024-12-16T07:22:27.896+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_9843500346_2024-12-16T07-22-22-743313Z', 'wait_until_call', 'scheduled__2024-12-16T07:22:22.743313+00:00', '--job-id', '529', '--raw', '--subdir', 'DAGS_FOLDER/complete_dag.py', '--cfg-path', '/tmp/tmp7my9b8id']
[2024-12-16T07:22:27.896+0000] {standard_task_runner.py:105} INFO - Job 529: Subtask wait_until_call
[2024-12-16T07:22:27.918+0000] {task_command.py:467} INFO - Running <TaskInstance: call_9843500346_2024-12-16T07-22-22-743313Z.wait_until_call scheduled__2024-12-16T07:22:22.743313+00:00 [running]> on host 1868de310944
[2024-12-16T07:22:27.952+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_9843500346_2024-12-16T07-22-22-743313Z' AIRFLOW_CTX_TASK_ID='wait_until_call' AIRFLOW_CTX_EXECUTION_DATE='2024-12-16T07:22:22.743313+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-16T07:22:22.743313+00:00'
[2024-12-16T07:22:27.953+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-16T07:22:27.954+0000] {logging_mixin.py:190} INFO - 126
[2024-12-16T07:25:27.954+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-16T07:25:27.961+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-16T07:25:27.961+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=call_9843500346_2024-12-16T07-22-22-743313Z, task_id=wait_until_call, run_id=scheduled__2024-12-16T07:22:22.743313+00:00, execution_date=20241216T072222, start_date=20241216T072227, end_date=20241216T072527
[2024-12-16T07:25:27.986+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-16T07:25:27.997+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-16T07:25:27.998+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
